### DISCUSSION TITLE
How can we make this first draft better?  We need your ideas!

### DISCUSSION THREAD

**Discussion by AndreaDowning** [TheLightCollective/PatientLedAI#2] [Ideas]
We're looking for feedback on how to word this draft better, and have public comments open for the next 30 days.

**Comment by SUSUGRUE** [comment]
I think this is a wonderful draft. Within Patient-Led Governance and Transparency, I think specific examples of actions to including patient input on the design, implementation, and use of AI applications could be beneficial. This could manifest in the form of linking to a roadmap with best practice for organizations to consider when including patient voice. I think usability of this document and the organizations perceived efficacy would be improved by linking to the step-by-step processes, strategies, and best practices for integrating patients' voices, rights, and representation throughout the AI development lifecycle in healthcare. For example, provide linkage to examples of  how to effectively include patients at every step or the best practice for doing so.
